docs/4to5.md

```diff
@@ -0,0 +1,82 @@
+# Jedis 4 to Jedis 5 Breaking Changes
+
+- `StreamConsumersInfo` has been renamed to `StreamConsumerInfo`.
+
+- Return type of `getAgeSeconds()` in `AccessControlLogEntry` has been changed from `String` to `Double`.
```

Comment:
For all of these I think you should change to 
```getAgeSeconds() in AccessControlLogEntry``` now returns `Double` instead of `String` .

Then, we can sort alphabetically too - and make it easier for users.